requirements/aws.txt

```diff
@@ -2,4 +2,4 @@
 boto3==1.17.36
 django-sslify==0.2.7
 django-ses==3.5.0
-uWSGI==2.0.22
+uWSGI==2.0.22; sys_platform != "win32"
```

Comment:
so does windows not need this to run?

Reply:
Not only does it not need it, they don't _have_ a Windows version, so it always fails with `pip install`